storage-proofs/src/sector_id.rs

```diff
@@ -0,0 +1,30 @@
+use byteorder::ByteOrder;
+
+#[derive(Debug, Clone, Copy, PartialOrd, PartialEq, Eq)]
+pub struct SectorId(u64);
+
+impl From<u64> for SectorId {
+    fn from(n: u64) -> Self {
+        SectorId(n)
+    }
+}
+
+impl From<SectorId> for u64 {
+    fn from(n: SectorId) -> Self {
+        n.0
+    }
+}
+
+impl std::cmp::Ord for SectorId {
```

Comment:
you should be able to derive `PartialOrd` and `Ord`

Reply:
Right you are! Updated.